tests/bin/prune-search-index_test.py

```diff
@@ -29,8 +41,8 @@ def add_stale_docs(self, path_list):
 
     def test_run_prune(self):
         path_list = [
-            "/old/artifact/old",
-            "/reallyold/artifact/ancient"
+            "/test/artifact/old",
```

Comment:
I believe the point of them being in different storage spaces/reference names/shelfs/buckets was to ensure that the utility could handle multiple buckets at once.